1.1/jessie/sdk/README.md

```diff
@@ -60,7 +56,7 @@ With this technique your application is compiled when you run `docker build` and
     FROM microsoft/aspnetcore-build
     WORKDIR /app
 
-    COPY project.json .
+    COPY MyApp.csproj .
```

Comment:
Not sure what I think about this, but would it make sense to use `*.csproj` here as a way to abstract the Dockerfile?